src/main/java/org/jabref/gui/util/component/TagBar.java

```diff
@@ -62,8 +63,10 @@ private void onTagsChanged(ListChangeListener.Change<? extends T> change) {
         while (change.next()) {
             if (change.wasRemoved()) {
                 tagList.getChildren().subList(change.getFrom(), change.getFrom() + change.getRemovedSize()).clear();
+                if(!allowsMultiple) { inputTextField.setDisable(false);}
```

Comment:
I think it is a bit easier to understand if you add the logic to enable / disable the input field after the while loop. I guess `inputTextField.setDisable(!tags.isEmpty())` should work.